### PR TITLE
Replace `with_argparser_and_unknown_args` in docs

### PR DESCRIPTION
**Pull request by KyleKing**
I replaced `with_argparser_and_unknown_args` with `with_argparser(parser, with_unknown_args=True)` per https://github.com/python-cmd2/cmd2/pull/1011#pullrequestreview-521180792

Let me know if you want me to make any follow-up changes!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,9 +123,8 @@ Instructions for implementing each feature follow.
     example in conjunction with the [conditional.py](https://github.com/python-cmd2/cmd2/blob/master/examples/scripts/conditional.py) script
 
 - Parsing commands with `argparse`
-    - Two decorators provide built-in capability for using `argparse.ArgumentParser` to parse command arguments
-        - `cmd2.with_argparser` - all arguments are parsed by the `ArgumentParser`
-        - `cmd2.with_argparser_and_unknown_args` - any arguments not parsed by the `ArgumentParser` get passed as a list
+    - The built-in `cmd2.with_argparser` decorator will parse arguments using `argparse.ArgumentParser`
```
Comment (tleonhardt) [inline README.md:126]: Changes LGTM

**Review by tleonhardt — APPROVED** [review]
Kyle, thank you for helping improve the documentation!

File: README.md (hunk 1 above)
Comment (KyleKing) [reply README.md:126]: Did you want me to make changes to this wording?

Comment (tleonhardt) [reply README.md:126]: Nope.   That was just an overall comment saying your changes look good to me.  I’ll give it a day or so to see if anyone else has any comments and then merge it in.

**Pull request merged**